index.html

```diff
@@ -4,9 +4,44 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://maple-ui.netlify.app/CSS/base.css">
+    <link rel="stylesheet" href="./CSS/quiz-home.css">
     <title>Document</title>
 </head>
 <body>
-    
+    <nav class="nav-bar">
+        <div class="nav-left">
+            <a class=" txt-xl semibold-font header-text-color" href="#">Maple Quiz</a>
+        </div>
+        <div class="nav-right">
+            <a href="" class=" txt-lg header-text-color">Docs</a>
```

Comment:
docs in quiz app?

Reply:
That was just for sample will update it thanks for pointing out